apps/frontend/src/modules/projects/projects-api.ts

```diff
@@ -28,6 +33,21 @@ class ProjectApi extends BaseHTTPApi {
 
 		return await response.json<ProjectGetAllResponseDto>();
 	}
+
+	public async getById(
+		payload: ProjectFindRequestDto,
```

Comment:
rename it to ProjectGetByIdRequestDto
```suggestion
		payload: ProjectGetByIdRequestDto,
```